modules/howtos/examples/managing_connections.py

```diff
@@ -16,16 +25,16 @@ def test_simpleconnect(self):
       # if your version of Couchbase Server supports this feature.
       customer_a = bucket.scope("customer-a")
       widgets = customer_a.collection("widgets")
-
       #end::simpleconnect[]
 
       # For a graceful shutdown, disconnect from the cluster when the program ends.
       cluster.disconnect()
 
 
     def test_multinodeconnect(self):
+      print("multinodeconnect")
       #tag::multinodeconnect[]
-      cluster = Cluster.connect("couchbase://192.168.56.101,192.168.56.102", ClusterOptions(PasswordAuthenticator("username", "password")))
+      cluster = Cluster.connect("couchbase://node1.example.com,node2.example.com", ClusterOptions(PasswordAuthenticator("username", "password")))
```

Comment:
This is really useful. Wasn't sure how we could test multi-connection examples for automation but seems this covers it! 👍